scripts/dashboard/Activity.vue

```diff
@@ -228,6 +233,13 @@ async function parseTXs(arrTXs) {
 
     txs.value = newTxs;
 }
+if (props.rewards) {
```

Comment:
maybe move this to the function `update` instead of `parseTXs`?

Reply:
It's not in `parseTXs`, it's at the top level

Reply:
oooh ok missed that